reference/best_practises/common_best_practises.rst

```diff
@@ -0,0 +1,26 @@
+Common Best Practises
```

Comment:
After re-check, "practice" as the noun, "to practice" is the verb, you're "practising" is the only usage of "s" instead of "c".